flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/artifact/FileSystemBasedArtifactFetcher.java

```diff
@@ -39,6 +39,7 @@ public File fetch(String uri, File targetDir) throws Exception {
         FileSystem fileSystem = source.getFileSystem();
         String fileName = source.getName();
         File targetFile = new File(targetDir, fileName);
+        LOG.info("Coping file from {} to {}", source, targetFile);
         try (var inputStream = fileSystem.open(source)) {
             FileUtils.copyToFile(inputStream, targetFile);
         }
```

Comment:
I think the line below this already log this. It's in `debug` level.

Reply:
removed and leave the others as `debug` also